webhook_server/tests/test_safe_rotating_handler.py

```
@@ -0,0 +1,192 @@
+                original_exists = os.path.exists
+
+                def mock_exists(path: str) -> bool:
+                    if path.endswith(".1"):
+                        return True  # Pretend .1 exists
+                    return original_exists(path)
+
+                original_remove = os.remove
+
+                def mock_remove(path: str) -> None:
+                    if path.endswith(".1"):
+                        raise FileNotFoundError(f"No such file: {path}")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)